SensusUI/PromptForInputsPage.cs

```diff
@@ -83,6 +83,24 @@ public PromptForInputsPage(InputGroup inputGroup, int stepNumber, int totalSteps
 
             int viewNumber = 1;
             bool anyRequired = false;
+
+            if (inputGroup.Inputs.Any(input => input.Display && input.GetView(viewNumber) != null && input.Required))
+            {
+                anyRequired = true;
+            }
+
+            if (_displayedInputCount > 0)
```

Comment:
I don't think `_displayedInputCount` will ever be non-zero here. Confirm and delete if so.